### ISSUE TITLE
Expend use of Group for metadata

### ISSUE THREAD

**Issue by vturpin** [OceanGlidersCommunity/OG-format-user-manual#106]
Moderator: @OceanGlidersCommunity/format-maintainers

Can we expend the use of groups to 

-  DEPLOYMENT_INFORMATION
-  PLATFORM_INFORMATION
-  FIELD_COMPARISON_INFORMATION
-  HARDWARE_INFORMATION
-  TELECOM_INFORMATION

Currently, those information are stored in variable. This would be coherent with the introduction of group in the format.

**Comment by castelao** [comment]
Yes, I agree. I've been thinking about that. What is not clear to me yet is what would be the best structure and groups. There is no limitation on the number of groups to be used, but we might want to use no more than necessary. Did you have a structure in mind?

**Comment by vturpin** [comment]
I agree, I would keep it simple.
I see two options : 
Keeping it like this. 5 extra groups. 
Or : 
We could regroup DEPLOYMENT* and FIELD* under group DEPLOYMENT
And regroup PLATFORM, HARDWARE and TELECOM under HARDWARE.
That would be 2 extra groups.

We could consider a more hierarchical organisation but I think it is too early.

**Comment by castelao** [comment]
Somehow I see all those in one single logical group. There is no hardware change once a deployment is initiated. But some failures could result in a hardware swap and re-deploy. Maybe the platform might be at a higher level, but I have the impression that all the rest would be within a deployment.
